src/chttpd_sup.erl

```diff
@@ -57,10 +54,8 @@ lru_opts() ->
             []
     end.
 
-auth_cache_handler() ->
-    case application:get_env(chttpd, auth_cache) of
-        {ok, Module} ->
-            Module;
-        _ ->
-            chttpd_auth_cache
-    end.
+add_auth_cache_if_missing(Children) ->
```

Comment:
Implemented in https://github.com/apache/couchdb-couch-epi/commit/33e8bae844e0994f024ee067196161d61872fcfb